docs/src/main/asciidoc/container-image.adoc

```diff
@@ -69,69 +72,46 @@ The creation of such objects is being taken care of by the Quarkus kubernetes ex
 
 == Building
 
-To build a container image for your project, you just need to specify `quarkus.container-image.build=true` either to the `application.properties` or as a system property.
+To build a container image for your project, `quarkus.container-image.build=true` needs to be set using any of the ways that Quarkus supports.
 
 [source, subs=attributes+]
 ----
 mvn clean package -Dquarkus.container-image.build=true
 ----
 
-Setting the environment variable `QUARKUS_CONTAINER_IMAGE_EXECUTION` to `build` can be used instead of the system propery.
-
 == Pushing
 
-If the execution is set to `push` then after the build, the image will also be pushed to an image registry.
-The registry can be specified using `quarkus.container-image.registry` and will default to `docker.io`.
+To build a container image for your project, `quarkus.container-image.push=true` needs to be set using any of the ways that Quarkus supports.
+
+[source, subs=attributes+]
+----
+mvn clean package -Dquarkus.container-image.push=true
```

Comment:
```suggestion
./mvnw clean package -Dquarkus.container-image.push=true
```

Reply:
We can use the wrapper in the Pushing section too, https://github.com/quarkusio/quarkus/blob/f16f9c9e208ab7d127d8851e8ac54633cc7df857/docs/src/main/asciidoc/container-image.adoc#pushing